pkg/services/featuremgmt/service.go

```diff
@@ -80,3 +81,8 @@ func ProvideManagerService(cfg *setting.Cfg, licensing licensing.Licensing) (*Fe
 func ProvideToggles(mgmt *FeatureManager) FeatureToggles {
 	return mgmt
 }
+
+// Same as ProvideToggles but satisfying the interface in plugins package
+func ProvidePluginsToggles(mgmt *FeatureManager) plugins.FeatureToggles {
```

Comment:
This is for feedback I receive from @wbrowne. In theory, `pkg/plugins` (generic), should not have a dependency from `pkg/servies` (implementation). In this case, I am not sure if it would be fine since we are just using the interface. @wbrowne opinions?

Reply:
We could introduce a specific plugins feature toggle service in `pkg/services/pluginsintegration/features` that implements the plugins interface by taking `featuremgmt.FeatureToggles`/`&featuremgmt/FeatureManager` as a dependency

Reply:
But that would still require a duplicated interface `plugins.FeatureToggles` (and its `Provider` function), no?

Reply:
Yes true but it addresses Marcus' concern by remove the dependency between freaturemgmt and plugins. The fact that it's duplicate is okay - it gives us the freedom to change the API if it makes sense.

It looks like we could also just rely on Wire binding to get  `plugins.FeatureToggles` to be implemented by `*featuremgmt/FeatureManager`, but I think having its own provider is probably best.

Reply:
well, the comment below refers to that

> Related to other comment. Can't you just use featuremgmt.ProvideToggles here since featuremgmt.FeatureToggles interface is satisfied by plugins.FeatureToggles?

Also:

> It looks like we could also just rely on Wire binding to get plugins.FeatureToggles to be implemented by *featuremgmt/FeatureManager, but I think having its own provider is probably best.

It needs its own provider (at least when I tried), it was not able to auto resolve `plugins.FeatureToggles`

Reply:
nevermind, checking this again I am able to not set the duplicated provider, which is preferrable (at least IMO)

Reply:
Oh nice 👍 Yeah that may be okay I think